gridt/src/app/home/home.page.ts

```diff
@@ -26,7 +26,7 @@ export class HomePage implements OnInit, OnDestroy {
     this.api.getSubscriptions();
   }
 
-  ngOnDestroy() { }
+  ngOnDestroy() {this.alertCtrl.dismiss(); }
```

Comment:
It is neater to do not use oneliners, as future git diffs will have to delete the entire line and replace it, while not actually changing the code. It is less readable as well.

Reply:
so would this be better?
![image](https://user-images.githubusercontent.com/17271270/92466477-706e7480-f1d0-11ea-84a1-532f83ca9b64.png)
or am i misunderstanding you?

Reply:
That is how I would like it.